juniper/Cargo.toml

```
@@ -6,6 +6,7 @@ edition = "2018"
 
 [dependencies]
 actix-web = "3"
+actix-cors = "0.3.0"
```

0.4.0 was released today

```suggestion
actix-cors = "0.4.0"
```